pkl-gradle/src/test/kotlin/org/pkl/gradle/TestsTest.kt

```diff
@@ -189,15 +254,15 @@ class TestsTest : AbstractTest() {
       .isEqualTo(
         """
       <?xml version="1.0" encoding="UTF-8"?>
-      <testsuite name="test" tests="6" failures="4">
-          <testcase classname="test" name="sum numbers"></testcase>
-          <testcase classname="test" name="divide numbers"></testcase>
-          <testcase classname="test" name="fail">
+      <testsuite name="test" tests="6" failures="3">
+          <testcase classname="test.facts" name="sum numbers"></testcase>
+          <testcase classname="test.facts" name="divide numbers"></testcase>
+          <testcase classname="test.facts" name="fail">
               <failure message="Fact Failure">4 == 9 ❌ (file:///file, line x)</failure>
               <failure message="Fact Failure">&quot;foo&quot; == &quot;bar&quot; ❌ (file:///file, line x)</failure>
           </testcase>
-          <testcase classname="test" name="user 0"></testcase>
-          <testcase classname="test" name="user 1">
+          <testcase classname="test.examples" name="user 0"></testcase>
+          <testcase classname="test.examples" name="user 1 #0">
```

Comment:
This looks much nicer! I like that we are giving information on whether this is a fact or an example.